source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,193 @@
+.. _c-get-started-download-and-install:
+
+====================
+Download and Install
+====================
+
+Download the {+driver-short+}
+----------------------
+
+The {+driver-short+} consists of two libraries: ``libbson`` and
+``libmongoc``. In this guide, you will learn how to install ``libmongoc``,
+which comes bundled with ``libbson``.
+    
+You can install the {+driver-short+} by using
+a package manager, or by installing the libraries from source. To learn
+how to install by using a package manager, select the tab corresponding to
+your operating system:
+
+.. tabs::
+
+   .. tab:: Linux
+      :tabid: install-linux
+
+      .. _c-redhat-install:
+
+      RedHat Enterprise Linux Distributions
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      You can install the driver on RedHat Enterprise Linux (RHEL)
+      distributions, version 7 or later, or distributions based on RHEL,
+      such as:
+
+      - `CentOS <https://www.centos.org/>`__
+      - `Rocky Linux <https://rockylinux.org/>`__
+      - `AlmaLinux <https://almalinux.org/>`__
+
+      Before you install the driver on a RHEL distribution, not including
+      Fedora, you must first enable the Extra Packages for Enterprise
+      Linux (EPEL) repository. You can enable the EPEL repository by
+      installing the ``epel-release`` package, as shown in the following
+      example:
+
+      .. code-block:: bash
+
+          yum install epel-release
+
+      Then, you can install the {+driver-short+}
+      by using ``yum``. To do so, run the following command in
+      your shell:
+
+      .. code-block:: bash
+
+         yum install mongo-c-driver-devel
+        
+      The preceding code example installs the developer packages, which
+      are required to build applications that use the {+driver-short+}. To
+      install only the libraries, without development files, run the
+      following command:
+
+      .. code-block:: bash
+
+         yum install mongo-c-driver
```

Comment:
Q: Should this be in a note admonition box? The way this reads makes this seems like an alternative to the steps above, so it could be helpful to separate it visually.

Reply:
Good call. I put this in an admonition and took the paragraph directly following out of the admonition to avoid having one right after the other (and I think it still works outside of one)